webots_ros2_core/webots_ros2_core/devices/device_manager.py

```diff
@@ -36,9 +37,10 @@ class DeviceManager:
     """Discovers Webots devices and creates corresponding ROS2 topics/services."""
 
     def __init__(self, node, config=None):
-        self._node = node
-        self._devices = {}
-        config = config or {}
+        self.__node = node
+        self.__devices = {}
+        self.__config = config or {}
+        self.__wb_devices = {}
```

Comment:
Why are you using sometimes single underscore and sometimes double underscores?

Reply:
I use single underscore for "protected" members and double for "private" members. Although, I am not sure if it is a good convention.

Reply:
looks indeed like a pretty standard convention, thank you I didn't knew about this subtility.